src/sagemaker/session.py

```diff
@@ -4273,10 +4284,38 @@ def get_model_package_args(
             metadata properties (default: None).
         domain (str): Domain values can be "COMPUTER_VISION", "NATURAL_LANGUAGE_PROCESSING",
             "MACHINE_LEARNING" (default: None).
+        sample_payload_url (str): The S3 path where the sample payload is stored (default: None).
+        task (str): Task values which are supported by Inference Recommender are "FILL_MASK",
+            "IMAGE_CLASSIFICATION", "OBJECT_DETECTION", "TEXT_GENERATION", "IMAGE_SEGMENTATION",
+            "CLASSIFICATION", "REGRESSION", "OTHER" (default: None).
+        framework (str): Machine learning framework of the model package container image
```

Comment:
The rest of these args are missing from the constructor inputs. Please remove them from the doc string

Reply:
Removed from doc string.